launcher/image/test/test_hardened_cloudbuild.yaml

```diff
@@ -1,35 +1,35 @@
 substitutions:
   # Expects hardened image (not debug) and should have startup-script service
   # disabled. google-startup-scripts.service is only enabled with multi-user.target.
-  '_IMAGE_NAME': 'confidential-space-51031c1-dev-hardened'
+  '_IMAGE_NAME': ''
   '_BASE_IMAGE_PROJECT': 'confidential-space-images-dev'
-  '_METADATA_FILE': 'startup-script=data/echo_startupscript.sh'
+  '_METADATA_FILE': 'startup-script=launcher/image/test/data/echo_startupscript.sh'
```

Comment:
let's keep this relative to the `launcher/image/test` dir. in the top level cloudbuild, you can `cd` into `launcher/image/test`.

same with the instances below.